src/styles/product_card.scss

```diff
@@ -0,0 +1,118 @@
+
+$cardWidth: 200px;
+$gapX: 40px;
+$paddingX: 40px;
+$colums: 4;
+
+.maincon {
+  margin: auto;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  margin-top: 0;
+  padding: 46px 26px 46px 28px;
+  width: $colums * $cardWidth + ($colums - 1) * $gapX + 2 * $paddingX;
+
+  &__product-card {
+    margin-right: 48px;
+    margin-bottom: 48px;
+
+    &:nth-child(4n) {
+      margin-right: 0;
+    }
+
+    &:nth-last-child(-n + 4) {
+      margin-bottom: 0;
+    }
+  }
+}
+
+.cards {
+  width: 200px;
+  height: 406px;
```

Comment:
Don't set fixed height to the card

Reply:
Set correct paddings to the card and use margin-top or margin-bottom to all card elements